src/systemd/sd-journal.h

```diff
@@ -72,6 +72,7 @@ enum {
         SD_JOURNAL_ALL_NAMESPACES            = 1 << 5, /* Show all namespaces, not just the default or specified one */
         SD_JOURNAL_INCLUDE_DEFAULT_NAMESPACE = 1 << 6, /* Show default namespace in addition to specified one */
         SD_JOURNAL_TAKE_DIRECTORY_FD         = 1 << 7, /* sd_journal_open_directory_fd() will take ownership of the provided file descriptor. */
+        SD_JOURNAL_FAST_QUERY                = 1 << 8, /* cache fstat() calls */
```

Comment:
Please make the name more descriptive and specific, e.g. `SD_JOURNAL_CALL_FSTAT_ONLY_ONCE` or so.

Reply:
A more descriptive term would be `SD_JOURNAL_CACHE_FILES_NEWEST_TIMESTAMPS`. The reason is the caching is applied way before `fstat()` is called, to cache the newest timestamps of journal files. There is no `fstat()` caching on this PR.

What would you prefer?

Reply:
> `SD_JOURNAL_CACHE_FILES_NEWEST_TIMESTAMPS`

Sounds good, though I have not followed the code in detail yet.